pkg/apis/trustmanager/v1alpha2/types_cluster_bundle.go

```diff
@@ -80,7 +80,8 @@ type BundleSpec struct {
 	InLineCAs *string `json:"InLineCAs,omitempty"`
 
 	// Target is the target location in all namespaces to sync source data to.
-	Target BundleTarget `json:"target"`
+	// +optional
+	Target *BundleTarget `json:"target,omitempty"`
```

Comment:
In https://github.com/cert-manager/trust-manager/pull/661, it seemed simpler to add the `omitzero` marker instead of making the field a pointer. Any thoughts here? Pointers makes things more complicated, but still considered best practice for optional struct fields AFAIK.

Reply:
I prefer to avoid pointers when it makes sense, and have now updated this PR with the same `omitzero` as in https://github.com/cert-manager/trust-manager/pull/661. It makes the conversion simpler and avoids potential nil pointers.